### PR TITLE
Styling refinements

### PR DESCRIPTION
**Pull request by aaronshekey**
This PR simplifies the app quite a bit, removing user input and just showing the Stacks browser support.

It also switches to webpack including Stacks, running purgecss, and then minifying. Neat!

image

### CODE REVIEW

**Review by israelpgjm — APPROVED** [review]

File: public/index.css [hunk 1]
```diff
@@ -1,12 +1,11 @@
+@import url(~@stackoverflow/stacks/dist/css/stacks.css);
+
+/* purgecss start ignore */
 .browser-image {
     background-position: center;
     background-size: contain;
 }
 
-.browser-image.internet-explorer {
```
Comment (b-kelly) [inline public/index.css:6]: Just at first glance, removing these concern me. They aren't used when parsing our default browser string, but could be used when parsing others. In addition, if we remove these from our css, we'd want to also remove these from our dependencies.

**Review by b-kelly — CHANGES_REQUESTED** [review]
At first look, I've spotted one blocking concern. Adding a review now to block the PR merge until I can look into it a little further.

File: public/index.css (hunk 1 above)
Comment (aaronshekey) [reply public/index.css:6]: Do we care all that much? I see this as less a general utility where users input their browserlist string, and more a fixed visual representation of Stacks' browserlist. My logic is that Stacks will never support Internet Explorer again.

If that's an unfair descoping of what this app does, I hear that. We could restore some user-input and these browser images down near the bottom of the page perhaps 🤔

**Comment by aaronshekey** [comment]
Merging as is to clean up the presentation. We can always revisit and restore the original functionality as needed :v:

**Pull request merged**